port/unix/omrsock.c

```diff
@@ -246,6 +279,62 @@ get_os_socket_option(int32_t socketOption)
 	return OMRPORT_ERROR_SOCK_OPTION_UNSUPPORTED;
 }
 
+/**
+ * @internal Map OMRSOCK API user interface socket flag to the OS socket, 
+ * flag which may be defined differently depending on operating system. 
+ *
+ * @param omrProtocol The OMR flag to be converted.
+ *
+ * @return OS socket flag on success, or 0 if none exists.
+ */
+static int32_t
+get_os_socket_flag(int32_t omrFlag)
+{
+	switch (omrFlag) {
```

Comment:
I think it would be a good idea to mask the omr flag here, rather than in the caller.